src/config/permissions.js

```diff
@@ -0,0 +1,15 @@
+const normal = {
+  articles: ['read', 'create', 'delete', 'edit'],
+  comments: ['read', 'create', 'delete', 'edit'],
+  tags: ['read', 'create', 'delete'],
+  users: ['read', 'create', 'edit', 'delete']
+};
+
+const admin = {
+  articles: ['read', 'delete'],
```

Comment:
it good to not allow an admin to create articles, I think an admin also can create an article

Reply:
For our case, the admin can only read and delete articles, but a normal user can create an article.